_posts/en/newsletters/2020-11-25-newsletter.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Bitcoin Optech Newsletter #125'
+permalink: /en/newsletters/2020/11/25/
+name: 2020-11-25-newsletter
+slug: 2020-11-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a website that tracks miner support for
+taproot activation and includes our regular sections with popular
+questions and answers from the Bitcoin StackExchange, announcements of
+releases and release candidates, and notable changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Website tracking miner support for taproot activation:** Bitcoin
```

Comment:
maybe s/tracking/signalling/ (to avoid the negative connotation of the word "tracking" mixed with "website", at least for me)

Reply:
No strong feelings about "tracking", but given the meaning of "signaling" in the context of soft fork activation, that might not be the best phrasing either.

Reply:
Ah oops, I only reviewed the first commit accidentally. "Listing" is a good choice!

Reply:
Good point about signalling; I didn't have a better idea but "listing" seems good and free from other connotations.